src/vs/code/electron-main/window.ts

```diff
@@ -29,6 +29,7 @@ export interface IWindowCreationOptions {
 	state: IWindowState;
 	extensionDevelopmentPath?: string;
 	allowFullscreen?: boolean;
+	macOSUseInlineToolbar: boolean;
```

Comment:
I am starting to think that we should make this more open ended because I also would like to have the option to have a custom dark title bar as a choice. This should be an enumeration with open ended values (e.g. default, inline, dark) so that we can continue with experiments.